lib/common/xml.c

```diff
@@ -2656,9 +2656,10 @@ pcmk__xc_update(xmlNode *parent, xmlNode *target, xmlNode *update)
  *                         child of this XML node that matches \p update
  * \param[in,out] target   If not NULL, update this XML
  * \param[in]     update   Make the desired XML match this (must not be NULL)
- * \param[in]     as_diff  If false, expand "++" when making attributes match
+ * \param[in]     as_diff  If false, expand PCMK__XML_PREFIX_CREATED when making
```

Comment:
This is an increment indicator rather than a patch indicator (e.g. if a="1", then someone sets a="a++", then a will end up as "2")

Reply:
Whoops, yep. I saw "diff" and assumed it was the diff prefix.